playwright/test/view-items.test.ts

```diff
@@ -206,41 +209,40 @@ test.describe('Scenario 5: A user wants to view an item in a different orientati
 });
 
 test.describe('Scenario 6: Item has multiple volumes', () => {
-  // TODO uncomment when e2es have been investigated further
-  // Worth noting that this one might genuinely be broken as it's the one that seems to always fail on desktop
-  // test('the volumes should be browsable', async ({ page, context }) => {
-  //   if (!isMobile(page)) {
-  //     await multiVolumeItem(context, page);
-  //     await page.waitForSelector(`css=body >> text="Volumes"`);
-  //     await page.click('text="Volumes"');
-  //     const navigationSelector = `nav [aria-label="${volumesNavigationLabel}"]`;
-  //     await page.waitForSelector(navigationSelector);
-
-  //     const navigationVisible = await page.isVisible(navigationSelector);
-  //     expect(navigationVisible).toBeTruthy();
-
-  //     const currentManifestLinkLabel = await page.textContent(
-  //       `${navigationSelector} a[aria-current="page"]`
-  //     );
-
-  //     const currentManifestLabel = await page.textContent(
-  //       '[data-test-id=current-manifest]'
-  //     );
-
-  //     expect(currentManifestLinkLabel).toEqual(currentManifestLabel);
-
-  //     const nextManifestLinkSelector = `${navigationSelector} a:not([aria-current="page"])`;
-  //     const nextManifestLinkLabel = await page.textContent(
-  //       nextManifestLinkSelector
-  //     );
-
-  //     await page.click(nextManifestLinkSelector);
-
-  //     await page.waitForSelector(
-  //       `css=[data-test-id=current-manifest] >> text="${nextManifestLinkLabel}"`
-  //     );
-  //   }
-  // });
+  test('the volumes should be browsable', async ({ page, context }) => {
+    if (!isMobile(page)) {
+      await multiVolumeItem(context, page);
+      await safeWaitForNavigation(page);
```

Comment:
This line is all I've added. My theory is that it used to fail because playwright goes too fast and starts the tests on the No JS version of the page, and then it changes over and it doesn't work.

Reply:
Oh yeah this is different! Back when we commented them, they just seemed flakey and sometimes passed.